src/parser/priest/holy/modules/core/EchoOfLightMastery.tsx

```diff
@@ -23,10 +24,14 @@ class EchoOfLightMastery extends Analyzer {
     combatants: Combatants,
   };
 
+  protected abilityTracker!: AbilityTracker;
+  protected healingDone!: HealingDone;
+  protected combatants!: Combatants;
+
   // All healing done by spells that can proc mastery
-  masteryHealingBySpell = {};
+  masteryHealingBySpell: any = {};
   // The eol pools currently on a target
-  targetMasteryPool = {};
+  targetMasteryPool: any = {};
```

Comment:
```typescript
targetMasteryPool: { 
  [targetId: number]: {
    pendingHealingTotal: number,
    pendingCritTotal: number,
    pendingHealingBySpell: {
      [spellId: number]: number
    },
    remainingTicks?: number,
    applicationTime?: number,
    pendingCritTotal?: number,
  }
} = {};
```

This one will likely create some errors, due to the last three values only sometimes being present. The best way to solve that depends on the actual logic behind how EoL works, so I cant give the best solution :(